website/project/views/node.py

```diff
@@ -426,7 +426,6 @@ def configure_requests(node, **kwargs):
 # View Project
 ##############################################################################
 
-@process_token_or_pass
```

Comment:
I don't thinkkk this would be needed anymore - it seemed like it was only there for these registration approval-type cases.